apps/ledger-live-mobile/src/components/CustomImage/FramedImageWithLottie.tsx

```diff
@@ -0,0 +1,49 @@
+import AnimatedLottieView from "lottie-react-native";
+import React, { useContext } from "react";
+import FramedImage, {
+  transferLottieConfig,
+  Props as FramedImageProps,
+  ImageSourceContext,
+} from "./FramedImage";
+
+const lottiesStyle = {
+  width: transferLottieConfig.frameWidth,
+  transform: [{ scale: (1182 / 502) ** (1 / 2) }], // (width of the lottie bounding box / width of the device frame in the lottie) ** (1 / 2)
```

Comment:
maybe we should put in a const somewhere? not sure if something can / should be reused